src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { Filters } from '../../types/Filters';
+
+type Props = {
+  activeTodos: Todo[],
+  complitedTodos: Todo[],
```

Comment:
There is a spelling mistake in 'complitedTodos', it should be 'completedTodos'. Please correct it to maintain the proper conventions and readability of the code.